content/spin/serverless-ai-hello-world.md

```diff
@@ -0,0 +1,396 @@
+date = "2023-10-20T16:00:00Z"
+title = "Build your first AI app using Serverless AI Inferencing"
+description = "Getting Started with building your AI app in Python, Rust or TypeScript"
+template = "spin_main"
+tags = ["ai", "serverless", "getting started"]
```

Comment:
@sohanmaheshwar , need to add the following 

```
enable_shortcodes = true
```

Only with this the tab blocks work.

Reply:
Gotcha. Thanks mate.